frontend/dashboard/components/RepoList/ActionLinks.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, DropdownMenu } from '@digdir/design-system-react';
+import React, { useRef, useState } from 'react';
+import classes from './RepoList.module.css';
+import cn from 'classnames';
+import {
+  ExternalLinkIcon,
+  FilesIcon,
+  MenuElipsisVerticalIcon,
+  PencilIcon,
+} from '@navikt/aksel-icons';
+import { useTranslation } from 'react-i18next';
+import { getRepoEditUrl } from '../../utils/urlUtils';
+import type { Repository } from 'app-shared/types/Repository';
+import { MakeCopyModal } from '../MakeCopyModal';
+
+type ActionLinksProps = {
+  repo: Repository;
+};
+
+export const ActionLinks = ({ repo }: ActionLinksProps): React.ReactElement => {
+  const { t } = useTranslation();
+  const [copyCurrentRepoName, setCopyCurrentRepoName] = useState('');
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const copyModalAnchorRef = useRef(null);
+
+  const handleOpenCopyModal = (repoFullName: string) => {
+    setModalOpen(true);
+    setCopyCurrentRepoName(repoFullName);
+  };
+
+  const handleCloseCopyModal = () => {
+    setModalOpen(false);
+    setCopyCurrentRepoName(null);
+  };
+
+  const repoFullName = repo.full_name;
+  const [org, repoName] = repoFullName.split('/');
+  const isDatamodelling = repoFullName.endsWith('-datamodels');
+  const editUrl = getRepoEditUrl({ org, repo: repoName });
+  const editTextKey = t(isDatamodelling ? 'dashboard.edit_datamodels' : 'dashboard.edit_app', {
+    appName: repoName,
+  });
+
+  return (
+    <div className={classes.actionLinksContainer} ref={copyModalAnchorRef}>
+      <Button variant={'tertiary'} className={classes.giteaButton} icon asChild>
+        <a
+          href={repo.html_url}
+          title={t('dashboard.show_repo', {
+            appName: repoName,
+          })}
+        >
+          <i className={cn('fa fa-gitea', classes.giteaIcon)} />
```

Comment:
What is 'fa fa-gitea'?

Reply:
It is how the Gitea icon is rendered. I don't know where it is from, I just copied it from the old implementation 😅